src/payment/strategies/masterpass/masterpass-payment-strategy.spec.ts

```diff
@@ -73,12 +74,15 @@ describe('MasterpassPaymentStrategy', () => {
         jest.spyOn(scriptLoader, 'load').mockReturnValue(Promise.resolve(masterpassScript));
         jest.spyOn(masterpassScript, 'checkout').mockReturnValue(true);
 
+        locale = 'en-US';
```

Comment:
Same here, instead of assigning to a temporary variable, you could directly pass it to the function